src/components/molecules/ExecutionDetails/TestExecutionDetails/TestExecutionDetailsTabs.tsx

```diff
@@ -83,7 +87,7 @@ const TestExecutionDetailsTabs: React.FC = () => {
     }
   }, [isRunning, id]);
 
-  const items = [
+  const defaultExecutionDetailsTabs = [
     {
       key: 'LogOutputPane',
       label: 'Log Output',
```

Comment:
Probably `Log Output` should receive high priority, maybe even `order: Infinity`

Reply:
`order` prop is not supported within the `Tab` interface, hence we'll need to extend the typing for this (and probably other usecases in the future). Isn't it enough if we can insert tabs from plugin in between the default once, without having to set order for the defaults? Only downside I see is, that we can't enforce a certain tab to ALWAYS be the first

Reply:
We cannot, as `order` will be unusable then. When you'll put `order: 1` to any element, it will go before every single one there. There is no way to order items *after* the defaults.

Reply:
Besides, `order` doesn't have to be a part of prop - it may be a metadata of the item spec, like `{ order: 3, item: { key: '', label: '' }}`